packages/runtime/test/src/runtime.ts

```diff
@@ -0,0 +1,88 @@
+import { createMsigAddress } from "@algorand-builder/algob";
+import { assert } from "chai";
+
+import { StoreAccount } from "../../src/account";
+import { Runtime } from "../../src/runtime";
+import { getProgram } from "../helpers/files";
+import { useFixture } from "../helpers/integration";
+
+const programName = "escrow.teal";
+const multiSigProg = "sample-asc.teal";
+
+describe("Logic Signature Test", () => {
+  useFixture("escrow-account");
+  const john = new StoreAccount(10);
+  const bob = new StoreAccount(100);
+  const runtime = new Runtime([john, bob]);
+
+  it("should sign the lsig by john(delegated signature)", () => {
+    const lsig = runtime.getLogicSig(getProgram(programName), []);
```

Comment:
why don't we use an SDK for that? SDK contains all functionality for creating LogicSig.

Reply:
same for multisig.

Reply:
the functionality that is offered by SDK takes compiled bytes of a program,
I wasn't sure if that will create any inconsistency, that's why i used `signBytes` and `verifyBytes` that signs bytes and verifies it. SDK functions need compiled bytes.
I will check again and see if we can use them without compiling program.